examples/jsm/loaders/IfcLoader/IfcLoader.d.ts

```diff
@@ -0,0 +1,14 @@
+import {
```

Comment:
Actually, we no longer include `d.ts` files in the repo, so this file can be removed.